docs/features.rst

```diff
@@ -23,66 +22,106 @@ For now, we extract the features for the whole recordings, and store them in sep
 We retrieve them by loading the whole feature matrix and selecting the relevant region (e.g. specified by a cut).
 Eventually we will look into optimizing the storage to further reduce the I/O.
 
-The feature matrices are accompanied by a YAML feature manifest. An example:
+There are two types of manifests:
+
+- one describing the feature extractor;
+- one describing the extracted feature matrices.
+
+The feature extractor manifest is mapped to a Python configuration dataclass. An example for *spectrogram*:
 
 .. code-block:: yaml
 
-    feature_extractor:
-      fbank_config:
-        use_log_fbank: true
-      mfcc_config:
-        cepstral_lifter: 22.0
-        num_ceps: 13
-      mfcc_fbank_common_config:
-        high_freq: 0.0
-        low_freq: 20.0
-        num_mel_bins: 23
-        use_energy: false
-        vtln_high: -500.0
-        vtln_low: 100.0
-        vtln_warp: 1.0
-      spectrogram_config:
-        dither: 0.0
-        energy_floor: 0.0
-        frame_length: 0.025
-        frame_shift: 0.01
-        min_duration: 0.0
-        preemphasis_coefficient: 0.97
-        raw_energy: true
-        remove_dc_offset: true
-        round_to_power_of_two: true
-        window_type: povey
-      type: fbank
-    features:
-      - channel_id: 0
-        duration: 16.04
-        num_features: 23
-        num_frames: 1604
-        recording_id: recording-1
-        start: 0.0
-        storage_path: test/fixtures/libri/storage/dc2e0952-f2f8-423c-9b8c-f5481652ee1d.llc
-        storage_type: lilcom
-        type: fbank
-
-The manifest consists of two parts:
-
-- a feature extractor configuration;
-- a list of documents describing each feature matrix.
-
-The feature extractor config is a detailed list of arguments used to extract the features in a given manifest.
-Currently, these arguments are the sum of all possible arguments for all feature types - we will likely refactor that
-in the future, and we might either move this section into a separate manifest or drop it entirely and store
-only the non-default values in each feature document.
-
-The feature documents contain the information necessary to tie them to a particular recording - ``start``, ``duration``,
+    dither: 0.0
+    energy_floor: 0.1
```

Comment:
Mm... if our signals are normalized to between -1 and 1, then I think energy-floor=0.1 is way too big.  This is appropriate in a Kaldi setting but not if we normalize the energy.

Reply:
I'll change to 1e-5

Reply:
should it be changed in this example (documentation) too?

Reply:
.. also I think 1e-10 would be more appropriate since it's the square of a magnitude.